### ISSUE TITLE
Extracted video stop automatically after every few seconds

### ISSUE THREAD

**Issue by Bhavesh-Multipz** [HaarigerHarald/android-youtubeExtractor#228]
Hello, I extracted a youtube video from a normal video and try to play it in Exo player, but it stops every few seconds, there is no connection issue, I connected with wifi. If anyone have solution please let me know, Thanks

**Comment by Mohaymin** [comment]
I'm facing similar issue
[Update] the video runs smoothly when it's loaded on webview. but this problem occurs in ExoPlayer.

**Comment by enfsol** [comment]
> I'm facing similar issue [Update] the video runs smoothly when it's loaded on webview. but this problem occurs in ExoPlayer.

Are you getting Video Quality Option on Webview ?? if yes could you please share how ?

**Comment by Mohaymin** [comment]
> 

No, there wasn't any video quality option. However, as far as I know, this option is not supposed to load with the extracted link.

**Comment by Mohaymin** [comment]
> > I'm facing similar issue [Update] the video runs smoothly when it's loaded on webview. but this problem occurs in ExoPlayer.
> 
> Are you getting Video Quality Option on Webview ?? if yes could you please share how ?

No, and as far as I know, the default media player of webview is not supposed to show video quality options with this extracted link.

**Comment by vivekAtHorizon** [comment]
does anyone of you have found the solution? I'm facing the same issue but with a Live youtube video instead of a normal youtube video
